src/pages/jiyoung/Login/Login.js

```diff
@@ -1,13 +1,102 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+import '../Login/Login.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
+
+  handleEmailInput = event => {
+    this.setState({
+      email: event.target.value,
+    });
+  };
+
+  handlePwInput = event => {
+    this.setState({
+      password: event.target.value,
+    });
+  };
+
+  handleSignUp = () => {
+    this.props.history.push('./main-jiyoung');
+    console.log(this.props.history);
+    // fetch('http://10.58.2.157:8000/users/login', {
+    //   method: 'POST',
+    //   body: JSON.stringify({
+    //     name: 'jiyoung',
+    //     email: this.state.email,
+    //     password: this.state.password,
+    //     phone_number: '123-456-7890',
+    //     bio: 'jiyoung_bio',
+    //   }),
+    // })
+    //   .then(res => res.json())
+    //   .then(res => {
+    //     console.log('결과: ', res);
+    //     this.setState({
+    //       commentList: res,
+    //     });
+    //
+    // if (res.token) {
+    //   localStorage.setItem('token', res.token);
+    //   this.props.history.push('/main-jiyoung');
+    // } else {
+    //   alert('아이디, 비밀번호 다시 입력해주세요.');
+    // }
+    //
+    //  });
+  };
+
   render() {
+    const { email, password } = this.state;
     return (
-      <div>
-        <h3>로그인</h3>
+      <div className="LoginJiyoung">
+        <div className="container">
+          <h1 lang="en">westagram</h1>
+          <div className="login-container">
+            <form action="" method="post" className="login-form">
+              <input
+                onChange={this.handleEmailInput}
+                type="text"
+                className="user-account"
+                placeholder="전화번호, 사용자 이름 또는 이메일"
+                required
+              />
+              <input
+                onChange={this.handlePwInput}
+                type="password"
+                className="user-pw"
+                placeholder="비밀번호(최소 5자 이상)"
+                required
+              />
+            </form>
+            <button
+              onClick={this.handleSignUp}
+              className={
+                email.includes('@') && password.length >= 5
+                  ? 'login-btn active'
+                  : 'login-btn'
+              }
+              disabled={
+                email.includes('@') && password.length >= 5 ? false : true
+              }
+            >
+              로그인
+            </button>
```

Comment:
같은 조건이 반복되고 있으니 해당 조건을 다음과 같이 변수로 선언 후 사용해보세요!
```
const condition = email.includes('@') && password.length >= 5;
```

```suggestion
            <button
              onClick={this.handleSignUp}
              className={condition ? 'login-btn active' : 'login-btn'}
              disabled={condition ? false : true}
            >
              로그인
            </button>
```

Reply:
className에 login-btn이 중복되고 있으니 좀 더 줄일 수도 있습니다!
참고해주세요!
```
className={`login-btn ${condition && 'active'}`}
```

Reply:
변수로 설정하거나 논리 연산자를 사용해 중복을 줄일 수 있는 방법 참고하겠습니다! 감사합니다:)